### PR TITLE
Issue #62

### PR DESCRIPTION
**Pull request by bazzooka**
Fix relating issue #62
Adding delay in animation did the trick. Only for FF when you have resized images in popover body.

### CODE REVIEW

**Comment by jasonkuhrt** [comment]
@bazzooka Thanks for putting this together! I'll try taking a look next week.

**Comment by bazzooka** [comment]
Very random bug :(
Thank :)

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "test": "testem",
     "start": "webpack-dev-server --hot",
     "prepublish": "npm run build",
+    "postinstall": "npm run build",
```
Comment (jasonkuhrt) [inline package.json]: Can you remove this please. Builds execute before publishing. We don't want to do this after installing.

File: lib/index.js [hunk 2]
```diff
@@ -296,8 +296,6 @@ const Popover = createClass({
     this.untrackPopover()
   },
   animateExitStop () {
-    clearTimeout(this.exitingAnimationTimer1)
-    clearTimeout(this.exitingAnimationTimer2)
```
Comment (jasonkuhrt) [inline lib/index.js]: I think this is wrong. These are `exiting` timers not `entering` which is what you removed below.

**Comment by jasonkuhrt** [comment]
I believe this was fixed by #64. Lets continue the discussion in #62 if not. Thanks!

**Pull request closed without merging**